src/Elements/ObjectDescription.php

```diff
@@ -6,9 +6,6 @@
 
 use PHPUnit\Architecture\Asserts\Properties\ObjectPropertiesDescription;
 
-/**
- * Open to extends
- */
-class ObjectDescription extends ObjectPropertiesDescription
+final class ObjectDescription extends ObjectPropertiesDescription
```

Comment:
Can you not make this class final? Or maybe simply add the `@final` annotation please?

Reply:
complete